src/util/ConfigManager/ConfigManager.cpp

```diff
@@ -25,11 +29,32 @@
 #include "util/ConfigManager/generated/ConfigShorthandLexer.h"
 #include "util/ConfigManager/generated/ConfigShorthandParser.h"
 #include "util/Exception.h"
+#include "util/Forward.h"
 #include "util/StringUtils.h"
 #include "util/antlr/ANTLRErrorHandling.h"
 #include "util/json.h"
 
 namespace ad_utility {
+static auto getDereferencedConfigurationOptionsViewImpl(
+    auto& configurationOptions) {
+  return std::views::transform(AD_FWD(configurationOptions), [](auto& pair) {
```

Comment:
Something that Sonarcloud found:
The forwarding here does nothing (you already only have the `auto&` as an argument, and anyway moving to a `view` does nothing. 
so you can simply drop the `AD_FWD` and make sonarcloud happy.

Reply:
Done, see [b8114e0](https://github.com/ad-freiburg/qlever/pull/1040/commits/b8114e0ea363681584329544f8e609adee1c1d15).